test/export/test_export.py

```diff
@@ -267,8 +275,56 @@ def body(x, y, z):
             return map(body, xs, y, z)
 
         inps = (torch.ones(6, 4), torch.tensor(5), torch.tensor(4))
-        exported_program = export(list_tensor_map, inps)
-        self.assertTrue(torch.allclose(exported_program(*inps), list_tensor_map(*inps)))
+        self._test_export_same_as_eager(list_tensor_map, inps)
+
+    def test_export_func_with_kwargs(self):
+        def kw_func(arg1, arg2, kw1, kw2):
+            return arg1 + arg2, kw1 + kw2
+
+        args = (torch.ones(6, 4), torch.ones(1, 1))
+        kwargs = {"kw1": torch.ones(1, 1), "kw2": torch.ones(6, 4)}
+        self._test_export_same_as_eager(kw_func, args, kwargs)
+
+    def test_export_func_with_pytree_kwargs(self):
+        def kw_func(arg1, arg2, a, b):
+            return arg1 + a["kw1"] + b[0], arg2 + a["kw2"] + b[1]
+
+        args = (torch.ones(2, 3), torch.ones(3, 4))
+        kwargs = {"a": {"kw1": torch.ones(2, 3), "kw2": torch.ones(3, 4)}, "b": [torch.ones(2, 3), torch.ones(3, 4)]}
+        self._test_export_same_as_eager(kw_func, args, kwargs)
+
+    def test_export_func_with_default_kwargs(self):
+        def kw_func(arg1, arg2, a, b=1):
+            return arg1 + arg2, a["kw1"] + a["kw2"] + b
+
+        args = (torch.ones(6, 4), torch.ones(1, 1))
+        kwargs1 = {"a": {"kw1": torch.ones(1, 1), "kw2": torch.ones(6, 4)}}
+        kwargs2 = {"a": {"kw1": torch.ones(1, 1), "kw2": torch.ones(6, 4)}, "b": 2}
```

Comment:
Could we add a test for what happens if you pass in "a" and then "b"?
```python
        def kw_func(arg1, arg2, a, b=1):
            return arg1 + arg2, a["kw1"] + a["kw2"] + b

        args = (torch.ones(6, 4), torch.ones(1, 1))
        kwargs = {"b": 2, "a": {"kw1": torch.ones(1, 1), "kw2": torch.ones(6, 4)}}
```

And for a test where "a" also has a default value but we only pass in "b"? 
```python
        def kw_func(arg1, arg2, a=1, b=2):
            return arg1 * a, arg2 * b

        args = (torch.ones(6, 4), torch.ones(1, 1))
        kwargs = {"b": 3}
```